accepted/00075-live-patching-enhancements.md

```diff
@@ -0,0 +1,189 @@
+
+ 1. Input a label prefix for the resulting filters. If the wizard is accessed from the project view, prefill this field with the project label
+ 2. Select a SLES client from a combobox
+ 3. Select a kernel version from a dropdown (current kernel version of the system is preselected)
+
+The resulting filters will set up the project for live patching from the specified kernel version.
+If the wizard is accessed from the project view, all the created filters will be automatically added to the project.
+
+#### Live patching for a product
+
+ 1. Input a label prefix for the resulting filters. If the wizard is accessed from the project view, prefill this field with the project label
+ 2. Select a synced SLES product from a dropdown
+ 3. Select a kernel version available in the selected product (latest is preselected)
+
+The resulting filters will set up the project for live patching from the specified kernel version.
+If the wizard is accessed from the project view, all the created filters will be automatically added to the project.
+
+#### Additional enhancements
+
+##### CLM filter list
+
+CLM filter wizards are aimed to make the project creation easier by automatically adding multiple filters. As a consequence, the list of filters in Uyuni can grow very quickly. To handle this problem, the following improvements can be made in the CLM filter list page:
+
+ - Add selection checkboxes to delete multiple filters at once
+ - Add "Delete unused/Select unused" shortcuts to delete or select all the filters that are not currently attached to any project
+ - Extend the search bar to search by project in use
+ - Make the "Project in use" column sortable
+
+##### Filter wizards for other purposes
+
+The concept of CLM filter wizards can be easily extended to other use cases as well. An example of such a use case is adding multiple AppStream module filters at once. A wizard called "AppStream modules with defaults" can be implemented with the following workflow:
+
+ 1. Input a label prefix for the resulting filters. If the wizard is accessed from the project view, prefill this field with the project label
+ 2. Select a modular channel from a dropdown
+ 3. In additional textboxes, add multiple module/stream pairs to override the default streams as required
+
+When done, the wizard creates a module filter per module in the repository, specifying the default stream for that module, or the specified stream if overridden in the wizard.
+
+# Drawbacks
+[drawbacks]: #drawbacks
+
+ - Gathering lifecycle data from the client is initially the easier option, but depending on clients is unnecessary and execution is duplicated per-client.
+
+# Alternatives
+[alternatives]: #alternatives
+
+Alternative implementations for EOL data retrieval are mentioned in the [data retrieval](#data-retrieval) section.
+
+# Unresolved questions
+[unresolved]: #unresolved-questions
+
+ - Which approach shall be used for [EOL data retrieval](#data-retrieval)?
```

Comment:
Third-one seems to be more complex but IMHO it makes the most sense and will be the most performant.